matlab/src/matlab/+arrow/+args/parseValidElements.m

```diff
@@ -0,0 +1,28 @@
+function validElements = parseValidElements(data, detectNulls, nullDetectionFcn)
+    % Returns a logical vector of the validElements in data. If inferNulls
```

Comment:
```suggestion
    % Returns a logical vector of the validElements in data. If detectNulls
```

A caller uses `inferNulls`. So we may want to rename `detectNulls` in this function to `inferNulls`...

Reply:
Good catch. I thought I renamed the parameter, but I guess I did not.

Reply:
I renamed the parameter to `inferNulls`.